packages/cli/src/workflow-execute-additional-data.ts

```diff
@@ -468,7 +468,7 @@ function hookFunctionsSave(): IWorkflowExecuteHooks {
 						(executionStatus === 'success' && !saveSettings.success) ||
 						(executionStatus !== 'success' && !saveSettings.error);
 
-					if (shouldNotSave) {
+					if (shouldNotSave && !fullRunData.waitTill) {
```

Comment:
Can we combine the conditions? `if (shouldNotSave && !fullRunData.waitTill && !isManualMode)`

Reply:
we could. but I honestly don't understand this code well enough, and just wanted to make quick fix. Maybe we could make this change when we finally start refactoring hooks?